pkgs/applications/misc/blender/wrapper.nix

```diff
@@ -8,7 +8,8 @@ stdenv.mkDerivation (finalAttrs: {
   pname = blender.pname + "-wrapped";
   src = blender;
 
-  inherit (blender) version meta;
+  inherit (blender) version;
+  meta = blender.meta // { mainProgram = pname; };
```

Comment:
I'm confused. How does this evaluate? There is no `pname` in scope, you'd needed to reference it as `finalAttrs.finalPackage.pname` or add `rec`/`with finalAttrs;` to make it work.

Reply:
it doesnt evaluate

```
blender.withPackages (p: [])
```

```

       at /home/kirillvr/.cache/nixpkgs-review/pr-268003/nixpkgs/pkgs/applications/misc/blender/wrapper.nix:12:42:

           11|   inherit (blender) version;
           12|   meta = blender.meta // { mainProgram = pname; };
             |                                          ^
           13|
```

Reply:
ah need to add rec